app/models/concerns/foreman_remote_execution/exportable.rb

```diff
@@ -0,0 +1,54 @@
+# This concern makes it easy to export an ActiveRecord object with specified
+# attributes and associations only.  If a specified assocation also includes
+# this concern, then it will likewise be exported. Custom attributes can be
+# specified as an options hast consisting of a custom export lambda.
+#
+# Example:
+#   attr_exportable :name, :address, :company => ->(user) { user.company.name }
+#
+module ForemanRemoteExecution
+  module Exportable
+    extend ActiveSupport::Concern
```

Comment:
I have no issues with this. Would there be any use case in other parts of Foreman for this? (I've got a feeling that there would be, although, we are using mainly the `rabl` to do this job)